.github/workflows/ci.yml

```
@@ -7,39 +7,89 @@ on:
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+
+      - name: Grant execute permission for Gradle
+        run: chmod +x ./gradlew
+
+      - name: Generate Jooq & Build Jar
+        run: ./gradlew clean generateJooq bootJar
+
+      - name: Upload Jooq Code
+        uses: actions/upload-artifact@v3
```

actionlint (1.7.7)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)